src/Cake.Common/Tools/NuGet/Add/NuGetAdder.cs

```diff
@@ -21,7 +21,7 @@ public sealed class NuGetAdder : NuGetTool<NuGetAddSettings>
         /// Initializes a new instance of the <see cref="NuGetAdder"/> class.
         /// </summary>
         /// <param name="fileSystem">The file system.</param>
-        /// <param name="environment">The environment.</param>
+        /// <param name="environment">The file system.</param>
```

Comment:
@phillipsj No worries, just ping us, when you fixed, rebased and squashed commits and we'll do a review again.